VendingMachine/VendingMachine.Core/RefundCalculator.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+
+namespace Vending.Core
+{
+    public class RefundCalculator
+    {
+        public IDictionary<Coin, int> CalculateRefund(int priceInCents, int paidInCents)
+        {
+            /* Note: I'm not particularly fond of the implementation here.
```

Comment:
Not a great comment. It doesn't add any value beyond an "I'm sorry" to the future maintainer.
It's best to just fix it.